src/routes/ViewAllLogs/ViewAllLogs.js

```diff
@@ -58,121 +54,49 @@ const {
 const INITIAL_RESULT_COUNT = 100;
 const RESULT_COUNT_INCREMENT = 100;
 
-const getQuery = (query, qindex) => {
-  if (query && qindex) {
-    return { [qindex]: logsSearchTemplate(query)[qindex] };
-  }
-
-  if (query) {
-    return logsSearchTemplate(query);
-  }
-
-  return {};
-};
-const getFilters = filters => {
-  const splitFiltersByGroups = () => {
-    const groups = {};
-
-    const fullNames = filters.split(FILTER_SEPARATOR);
-
-    for (const fullName of fullNames) {
-      if (fullName) {
-        const [groupName, fieldName] = fullName.split(FILTER_GROUP_SEPARATOR);
-
-        if (groups[groupName] === undefined) groups[groupName] = [];
-        groups[groupName].push(fieldName);
-      }
-    }
-
-    return groups;
-  };
-  const getMappedValues = (values, group) => {
-    return values.map(value => {
-      const obj = group.values.filter(f => typeof f === 'object' && f.name === value);
-
-      return (obj.length > 0) ? obj[0].cql : value;
-    });
-  };
-
-  if (filters) {
-    const groups = splitFiltersByGroups();
-    const filtersObj = {};
-
-    for (const groupName of Object.keys(groups)) {
-      const group = filterConfig.filter(g => g.name === groupName)[0];
-
-      if (group && group.cql) {
-        const cqlIndex = group.cql;
-
-        // values contains the selected filters
-        const values = groups[groupName];
-
-        const mappedValues = getMappedValues(values, group);
-
-        if (group.isRange) {
-          const { rangeSeparator = ':' } = group;
-          const [start, end] = mappedValues[0].split(rangeSeparator);
-
-          filtersObj.completedAfter = [start];
-          filtersObj.completedBefore = [end];
-        } else {
-          const {
-            noIndex,
-            values: groupValues,
-          } = group;
-
-          // fill in filters object
-          if (!noIndex) {
-            if (filtersObj[cqlIndex] === undefined) filtersObj[cqlIndex] = [];
-
-            filtersObj[cqlIndex] = [...filtersObj[cqlIndex], ...values];
-          } else {
-            values.forEach(value => {
-              const obj = groupValues.filter(f => typeof f === 'object' && f.name === value);
-
-              if (obj.length > 0) {
-                const groupIndex = obj[0].indexName;
-
-                if (filtersObj[groupIndex] === undefined) filtersObj[groupIndex] = [];
+const entityKey = 'jobLogs';
 
-                filtersObj[groupIndex] = [...filtersObj[groupIndex], ...obj[0].cql];
-              }
-            });
-          }
-        }
+export const ViewAllLogsManifest = Object.freeze({
```

Comment:
You don't need to move the manifest into a separate variable.
Refactor it next way:
1. Decorate ViewAllLogs using `@` symbol
2. Move manifest out of the class and define it like `ViewAllLogs.manifest = Object.freeze()`
3. Call manifest in tests like a static param - `ViewAllLogs.manifest`

Reply:
I tried this version but the browser could not render it in my case, could you check if it does in your environment.

Reply:
You're right, let's just leave it like that